src/commands/edge/drivers.ts

```diff
@@ -17,6 +22,15 @@ export default class DriversCommand extends EdgeCommand {
 		description: 'the driver id or number in list',
 	}]
 
+	static examples = [`# list all user-owned drivers
+$ smartthings edge:drivers
+
+# display details about the third driver listed in the above command
+$ smartthings edge:drivers 3
+
+# display details about a driver by using its id
+$ smartthings edge:drivers 699c7308-8c72-4363-9571-880d0f5cc725`]
+
```

Comment:
A small detail, but I noticed that OCLIF modifies the plural/singular of the output in the readme depending on how many items the examples array has. This one has `EXAMPLE` despite threre being multiple command examples. [Here](https://github.com/SmartThingsCommunity/smartthings-cli/blob/8c81c12a16b2739291f769fa40a997d296ff7d8e/packages/cli/src/commands/devicepreferences/translations.ts#L20) is an instance where we split some template strings into separate "Examples".

Reply:
I found a way for this to not look too bad and still do what we want. It's a little odd but it makes eslint happy, gets us the `S`, keeps things aligned as in the output, and has blank or nearly blank lines where there are blank lines in the output.